scripts/ubuntu-bionic/10-v4l2loopback.sh

```diff
@@ -15,7 +15,14 @@ fi
 
 apt-get install v4l2loopback-dkms -y
 
-if [[ "$CLOUD" == "google" ]]; then
+if [[ "$TASKCLUSTER_INSTANCE" == "firefoxci" ]]; then
+    # We must keep doing this for GCP workers at the moment.
+    V4L2LOOPBACK_VERSION=${V4L2LOOPBACK_VERSION:-0.12.5}
+    git clone -b v$V4L2LOOPBACK_VERSION https://github.com/umlaeute/v4l2loopback /usr/src/v4l2loopback-$V4L2LOOPBACK_VERSION
+    # Edit the file `v4l2looback.c` and change the `MAX_DEVICES` definition to `100`
+    # (NOTE: ignore the comments about overriding it in a `make` invocation; this isn't possible via dkms)
+    sed -i -e "s/# *define MAX_DEVICES 8/# define MAX_DEVICES $NUM_LOOPBACK_VIDEO_DEVICES/g" /usr/src/v4l2loopback-$V4L2LOOPBACK_VERSION/v4l2loopback.c
```

Comment:
Including the `8` in this `sed` is particular prone to silent breakage when not cloning from a frozen version of the repo.